### ISSUE TITLE
mermaid sequenceDiagram not working?

### ISSUE THREAD

**Issue by KGillinder** [rich-iannone/DiagrammeR#501]
Many of the features of mermaid sequenceDiagrams are not working? or perhaps they aren't supported?

```
library(DiagrammeR)
mermaid("
sequenceDiagram
    Alice->>Bob: Hello Bob, how are you ?
    Bob->>Alice: Fine, thank you. And you?
    create participant Carl
    Alice->>Carl: Hi Carl!
    create actor D as Donald
    Carl->>D: Hi!
    destroy Carl
    Alice-xCarl: We are too many
    destroy Bob
    Bob->>Alice: I agree
")
```

However, removing the 'create' and 'destroy' calls works.

Activations do not work. i.e., 

```
sequenceDiagram
    Alice->>John: Hello John, how are you?
    activate John
    John-->>Alice: Great!
    deactivate John
```

Additionally, there does not appear to be any way to format colours and fills for the components of the diagram?

**Comment by olivroy** [comment]
Hi! This was fixed in mermaid 8.1.0. https://github.com/mermaid-js/mermaid/issues/647 This would work if DiagrammeR updated its bundled mermaid.js version.

It caused breaking changes when it was first attempted. See #475 for future updates.

**Comment by rich-iannone** [comment]
Closing this in favor of https://github.com/rich-iannone/DiagrammeR/issues/475.
